Makefile

```diff
@@ -22,7 +22,7 @@ check:
 	$(CC) test -v ./...
 
 build:
-	env GOOS=$(OS) GOARCH=$(ARCH) GOARM=$(ARM) $(CC) build -o $(NAME)-$(OS)-$(ARCH) cmd/plugin.go
```

Comment:
can you restore this change? Thanks!

Reply:
@vincenzopalazzo I was trying to explain that removing those makes the plugin work but having it there the plugin doesn't work

Reply:
because you are build it of wrong architecture, you can build the plugin for debug with just `make`